src/limbo/model/gp/kernel_lf_opt.hpp

```diff
@@ -59,7 +59,7 @@ namespace limbo {
                         // K^{-1} using Cholesky decomposition
                         Eigen::MatrixXd w = Eigen::MatrixXd::Identity(n, n);
 
-                        gp.matrixL().template triangularView<Eigen::Lower>().solveInPlace(w);
+                        // gp.matrixL().template triangularView<Eigen::Lower>().solveInPlace(w);
```

Comment:
Ohh yes... You're right.. I didn't see the `solveInPlace` function.. I thought that we were doing the same thing twice.. Fixing it..